reader/lib/migrations/index.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+import { uniq, get } from 'lodash'
+import minimist from 'minimist';
+import cache from './cache.json';
+import config from './config';
+
+const argv = minimist(process.argv.slice(2));
+const dir = fs.readdirSync(path.join(__dirname, './lib'));
+
+const env = argv['env'] || 'local';
+
+const { ckanUrl, ckanToken } = config[env];
+const alreadyRun = get(cache, env, [])
+
+const promises = dir.filter(d => argv['force'] || !alreadyRun.includes(d)).map(async filename => {
```

Comment:
I think I get the cache now, it's functioning as a historical marker, so that we can noop previously run migrations? In that case is `history` a more suitable dir name? I might've gotten this wrong

Reply:
yes you're right, a naiive file to cache run migrations. I have seen `cache` use in similar implementations, but perhaps a moar descriptive name would be `run-migrations-cache` or `run-migrations-log`. History seems like the wrong word, but you know what they say about naming things.